tealium/volatiledata/TealiumVolatileData.swift

```diff
@@ -65,14 +68,18 @@ public class TealiumVolatileData {
     
     /// Auto reset the session id now.
     public func resetSessionId() {
+
         self.add(data:[TealiumVolatileDataKey.sessionId: newSessionId() ])
     }
```

Comment:
Are you referring to the return spacing between lines of code and the ending "}" or the space between newSessionId() and ]?